test/framework/src/main/java/org/elasticsearch/test/rest/TestFeatureService.java

```diff
@@ -8,8 +8,24 @@
 
 package org.elasticsearch.test.rest;
 
+import java.util.Set;
+
 public interface TestFeatureService {
     boolean clusterHasFeature(String featureId);
 
-    TestFeatureService ALL_FEATURES = ignored -> true;
+    Set<String> getAllSupportedFeatures();
+
+    TestFeatureService ALL_FEATURES = new TestFeatureService() {
```

Comment:
Does it make sense to have this defined here, now the interface is more complicated? It's only used in ESRestTestCase, and the implementation of `getAllSupportedFeatures` references its use. So I think this should instead be the initial instance of `ESRestTestCase.testFeatureService`

Reply:
That's a good point; I'll move it there.

Reply:
I was also wondering if I should rename it to something like `UNINITIALIZED`, but decided to leave it as-is for now